tools/tasks/seed/build.bundles.app.rollup.exp.ts

```diff
@@ -0,0 +1,43 @@
+import Config from '../../config';
+import { writeFile } from 'fs';
+import { join } from 'path';
+
+const nodeResolve = require('rollup-plugin-node-resolve');
+const commonjs = require('rollup-plugin-commonjs');
+const rollup = require('rollup');
```

Comment:
Not available, I didn't find type definitions for these. But if anyone knows where I can find the typing I can add them and use import instead. Since types info is not available I am forced to use require instead.